dlt/dlt.py

```diff
@@ -946,9 +963,18 @@ def connect(self, timeout=None):
                         self.sock = ctypes.c_int(self._connected_socket.fileno())
                         # - also init the receiver to replicate
                         # dlt_client_connect() behavior
-                        connected = dltlib.dlt_receiver_init(ctypes.byref(self.receiver),
-                                                             self.sock,
-                                                             DLT_CLIENT_RCVBUFSIZE)
+                        api_ver = tuple(int(num) for num in API_VER_STR.split('.'))
```

Comment:
This line duplicates several times. Could you create a new variable

```python
API_VER_TUPLE = tuple(int(num) for num in API_VER_STR.split('.'))
```

in `dlt/core/__init__.py` (after line 65)

Reply:
Done